index.html

```diff
@@ -13,7 +13,19 @@
 
 <body>
     <div class="container">
-   
+
+        <div class="nav-bar">
+            <div class="nav-logo">
+                <a href="index.html"><img src="img/lambda-black.png"></a>
+            </div>
```

Comment:
should work with or without the div like we went over. this way is fine as well, dealers choice =]